docs/schema/guidance/record-ids.rst

```diff
@@ -4,4 +4,53 @@
+
+Record identifiers are used for :ref:`representing-bo` and for :ref:`guidance-updating-data`.
+
+Example scenario
+----------------
+
+Private companies in a jurisdiction are obliged to disclose to the Company Registration Authority: their legal owners and any beneficial owners with direct interests in the company. 
+
+Alpha Corp. discloses one natural person as a beneficial owner (and legal owner) and it also discloses Beta Systems as a legal owner.
+
+.. figure:: ../../_assets/AlphaCorp.svg
+   :alt: Diagram of a beneficial ownership network. An entity node for the company Alpha Corp is connected via a solid line to a person node. Alpha Corp's node is also connected via a solid line to an entity node for the company Beta Systems. An entity statement block sits next to Alpha Corp's node. Within the entity statement, recordId is '04a221be'. An entity statement block sits next to Beta Systems's node. Within the entity statement, recordId is '09d3a4fb'. 
+   :figwidth: 80%
+   :align: center
+
+Beta Systems declares that it has four direct beneficial owners. 
+
+.. figure:: ../../_assets/BetaSys.svg
+   :alt: Diagram of a beneficial ownership network. An entity node for the company Beta Sys is connected via a solid lines to four person nodes. An entity statement block sits next to the Beta Sys node. Within the entity statement, recordId is '02c83aa2'.
+   :figwidth: 60%
+   :align: center
+
+Separate records are maintained by the Company Registration Authority for the information about Beta Systems disclosed by Alpha Corp. (record identifier 09d3a4fb) and the information which is self-disclosed by Beta Systems (record identifier 02c83aa2).
+
+When the BODS data is published, there are statements about the two new Beta Systems records, as below. 
+
+.. figure:: ../../_assets/AlphaBeta.svg
+   :alt: Diagram of two entity statement blocks. The first has: recordId '09d3a4fb', recordDetails.name 'BetaSys', recordDetails.identifiers.0.scheme 'TZ-CRA', recordDetails.identifiers.0.id: 'pc-239118274', declarationSubject: '04a221be'. The second has: recordId '02c83aa2', recordDetails.name 'Beta Systems', recordDetails.identifiers.0.scheme 'TZ-CRA', recordDetails.identifiers.0.id: 'pc-239118274', declarationSubject: '02c83aa2' 
+   :figwidth: 100%
+   :align: center
+
+Note that:
+
+- The name of the company has not been consistently disclosed. 
+- The two statements do pertain to the same real world entity, as the ``scheme`` and ``id`` properties in both statements are the same.
+- The two statements about the same real world entity relate to two different declaring subjects, as shown by the different ``declarationSubject`` values.
+- The statement for ‘Beta Systems’ has been disclosed by Beta Systems itself, since the ``recordId`` value is also given as the ``declarationSubject``.
```

Comment:
```suggestion
- the statement for ‘Beta Systems’ has been disclosed by Beta Systems itself, since the ``recordId`` value is also given as the ``declarationSubject``.
```